recipes/civetweb/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)
```

Comment:
You are missing a check in configure for the cppstd version if this is a requirement.

```py

if self.settings.compiler.cppstd:
    tools.check_min_cppstd(self, 14)

```

Reply:
Well, it is a strange requirement, because it is written in C. And it looks like this line is not needed in test_package's CMakeLists.txt. I will remote it.